content/user/ui/overview/Basic-quick-start-guide.md

```diff
@@ -0,0 +1,111 @@
+<!--
+title: "Quick Start Guide"
+description: "Overview of Contrast UI for new users"
+tags: "TeamServer user applications vulnerabilities"
+-->
+
+Welcome to Contrast! This is your guide to everything you need to know to get started in the interface. Start with onboarding your applications and then move on to environments, libraries and vulnerabilities – you’ll be up and running by the time you’re done. As always, if you have any more questions, let the Contrast Support team know by emailing <support@contrastsecurity.com>. 
+
+## Onboard Your Application
+
+Once you've logged onto Contrast, click the **Add Applications** button on the top right, which brings you to an agent download page. A wizard walks you through what needs to be done. What language is your application primarily in – Java, .NET or Node.js? If you're using Java, you'll notice that there are two options: Java and Java 1.5. The only difference is that Java includes the functionality to do automatic updates, which isn’t available in Java 1.5.
+ 
+When you download an agent, the file should be called *contrast.jar*. This file needs to stay in its current form and shouldn’t be renamed. Once you download the *contrast.jar* file, regardless of your application server, Contrast needs to pass in **-javaagent:/path/to/server/contrast.jar** to your application server's JVM. Once complete, restart your application server. 
+
+For more information on adding an application, please read the article on [Adding an Application](https://docs.contrastsecurity.com/user_tsguideapp.html#addapp).
+ 
+## Use the application
+
+The application server shows up right away in Contrast under the Servers tab. However, the application won’t appear in Contrast until you browse through it and generate some traffic. When an application first appears in Contrast, it’s listed as a trial application. As a trial application, the interesting information is blurred out under the application's Vulnerabilities tab. (Note that the same vulnerabilities appear on the Vulnerabilities page.) 
+
+To  see all the information, these applications must be licensed. For more information on licensing your application, read the article on [Licensing an Application](https://docs.contrastsecurity.com/user_tsguideapp.html#licapp).
+ 
+## Manage Applications
+
+Go to the Applications page to get detailed information, see findings, scores, manage licenses, settings and more.
+ 
+## Set Up Environments
```

Comment:
I still don't fully understand when we want to capitalize all, vs. use a sentence-style heading. "Use the application" vs. "Set Up Environment" and "How vulnerabilities work" vs. "Improve Your Application Score". Just noting here in case that needs to be reviewed again.